ssn/index.html

```diff
@@ -5283,91 +5283,91 @@ <h5><a href="#OMSsamplePreparationStep">sosa-oms:samplePreparationStep</a></h5>
         </section>
     <section id="OMS-Alignment-Class">
       <h4>Class Alignments</h4>
-      <p>The following classes in SOSA can be aligned as follows. </p>
+      <p>The following classes used in SOSA can be aligned as follows. </p>
       <table>
         <tbody>
           <tr>
-            <td> sosa:Deployment </td>
+            <td> <a href="SOSADeployment">sosa:Deployment</a> </td>
             <td> ogc-ms:implements </td>
             <td> obs-cpt:Deployment </td>
           </tr>
           <tr>
-            <td> sosa-oms:MaterialSample </td>
+            <td> <a href="SOSAoms:MaterialSample">sosa-oms:MaterialSample</a> </td>
```

Comment:
This `href` element is creating a different HTML hyperlink. Please verify.

Reply:
Fixed by https://github.com/w3c/sdw-sosa-ssn/commit/908d065c33a9d822003c7bdd8b5f116fae70f89e

Reply:
@ldesousa can you verify that the issue is resolved